index.html

```diff
@@ -24,22 +25,6 @@
                title="Bazel"
                alt="Bazel logo"
                class="logo img-responsive" />
-          <p class="tagline">{Fast, Correct} - Choose two</p>
```

Comment:
I love "{fast, correct} chose two" but this message fails to desvribe Bazel and what Bazel can do for the user.
Yes it's the Bazel tagline and we should keep using it.